framework/doc/content/syntax/Outputs/index.md

```diff
@@ -2,9 +2,336 @@
 
 # Outputs System
 
+- The output system is designed to be just like any other system in MOOSE: modular and expandable.
+- It is possible to create multiple output objects for outputting:
+
+  - at specific time or timestep intervals,
+  - custom subsets of variables, and
+  - to various file types.
+
+- Supports commnon parameters and sub-blocks.
+
+```puppet
+[Outputs]
+  file_base = <base_file_name> 
+  exodus = true
+[]
+```
+
+```puppet
+[Outputs] 
+  vtk = true 
+  [./console]
+    type = Console
+    perf_log = true
+    max_rows = true
+  [../]
+  [./exodus]
+    type = Exodus
+    execute_on = 'timestep_end'
+  [../] 
+  [./exodus_displaced]
+    type = Exodus
+    file_base = displaced
+    use_displaced = true
+    interval = 3
+  [../] 
+[]
+```
+
+## Outputs and execute_on
+
+Outputs utilize the "execute_on" parameter like other systems in MOOSE, by default:
+
+- All objects have `execute_on = 'initial timestep_end'`, with `Console` objects being the exception.
+- `Console` objects append `'initial timestep_begin linear nonlinear failed'` to the "execute_on" settings at the common level.
+
+When debugging output, use the `--show-outputs` flag when executing your application. This will add a section near the top of the simulation that shows the settings being used for each output object.
+
+## Basic Input File Syntax
+
+- To enable output an application it must contain an `[Outputs]` block.
+- The simplest method for enabling output is to utilize the short-cut syntax as shown below, which enables `Exodus` output for writing data to a file.
+
+```puppet
+[Outputs]
+  exodus = true   # output to ExodusII file with default settings
+[]
+```
+
+## Full Block Syntax
+
+- To take full advantage of the output system the use of sub-blocks is required.
+- These blocks are identical to the previous section, including the block names (i.e. the short-cut syntax builds these exact blocks).
+- The resulting filenames produced by different syntax is important and discussed in the File Output Names section.
+
+```puppet
```

Comment:
You can delete `puppet` from all these lines, the default syntax highlighting works fine for these cases now.